website/content/docs/auth/login-mfa.mdx

```diff
@@ -0,0 +1,170 @@
+  alias.
+
+- `Duo` - If Duo push is configured and enabled on a login path, then the enrolled
+  device of the user will get a push notification to approve or deny the access
+  to the API. The Duo username will be derived from the caller identity's
+  alias.
+
+- `PingID` - If PingID push is configured and enabled on a login path, then the
+  enrolled device of the user will get a push notification to approve or deny
+  the access to the API. The PingID username will be derived from the caller
+  identity's alias.
+
+## Login MFA Procedure
+
+Login MFA can be configured to further secure authenticating to an auth method. To enable login
+MFA, an MFA method needs to be configured. Please see [Login MFA API](/api/secret/identity/mfa) for details
+on how to configure an MFA method. Once an MFA method is configured, using the returned unique MFA method ID,
+an operator can configure an MFA enforcement. Please see [Login MFA Enforcement API](/api/secret/identity/mfa/login-enforcement)
+for details on how to configure an MFA enforcement config. MFA could be enforced for an entity, a group of
+entities, a specific auth method accessor, or an auth method type. A login request that matches
+either of the MFA enforcement restrictions is subject to further MFA validation,
+like a one-time passcode, before being authenticated.
+
+There are two ways to validate a login request that is subject to MFA validation.
+
+ ### Single-Phase Login
+ In the Single-phase login, the required MFA information is embedded in a login request using
+ the `X-Vault-MFA` header. In this case, the MFA validation is done
+ as a part of the login request.
+
+ MFA credentials are retrieved from the `X-Vault-MFA` HTTP header. The format of
+ the header is `mfa_method_id[:key[=value]]`. The item in the `[]` is optional. If there are more than
+ one MFA method that needs to be validated, a user can pass in multiple `X-Vault-MFA` HTTP headers.
+
+ #### Sample Request
+
+ ```shell-session
+ $ curl \
+     --header "X-Vault-Token: ..." \
+     --header "X-Vault-MFA:d16fd3c2-50de-0b9b-eed3-0301dadeca10:695452" \
+     http://127.0.0.1:8200/v1/auth/userpass/login/alice
+ ```
+
+ ### Two-Phase Login
+ The more conventional, and prevalent MFA method is a two request mechanism, also referred to as Two-phase Login MFA.
+ In Two-phase login the `X-Vault-MFA` header is not provided in the request. In this case, after sending a regular login request,
+ the user receives an auth response in which MFA requirements are included. MFA requirements contain an MFA request ID
+ which identifies the login request that needs to get validated. In addition, MFA requirements contain MFA constraints
+ that identify which MFA types should be used to validate the request, the corresponding method IDs, and
+ a boolean value showing whether the MFA method uses passcodes or not. MFA constraints form a nested map in MFA Requirement,
+ and represent all MFA enforcements that match a login request.
```

Comment:
Would something like this make sense, just to call out that we aren't updating all login docs?
`While the example below is for the userpass login, it should be noted that this can affect the login response on any auth mount protected by MFA validation`